jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -64,98 +62,22 @@ def __init__(self, *args, **kwargs):
         self.state.image_external_padding = 0.5
 
     def on_mouse_or_key_event(self, data):
-
-        # Find visible layers
-        visible_layers = [layer for layer in self.state.layers
-                          if (layer.visible and layer_is_image_data(layer.layer))]
-
-        if len(visible_layers) == 0:
-            return
```

Comment:
This check is needed also for other stuff in here that you are not touching.

Reply:
I think only `image = self.active_image_layer.layer` would be affected now since `self.active_image_layer` could technically return `None`.  So I can re-introduce something similar like

```
active_image_layer = self.active_image_layer
if active_image_layer is None:
    return
```

Reply:
I haven't seen the full diff, so if `self.active_image_layer` is the correct check now, please use that. The point is, the rest of the stuff here still need some sort of check.